Bottom: 8bc69cc85f5ae9c4288d7ebda349e8b41ca26782
Top:    4a201f03d97667877b3a9f5e9a6737bebe68fa31
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 04:26:08 +0000

Refresh of find-c-kill-setthing-with-fire

---

diff --git a/src/find.c b/src/find.c
index 18385d7..a93679d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3673,7 +3673,7 @@ DRCAll (void)
             SET_FLAG (flag, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetThing (LINE_TYPE, layer, line, line);
+            SetFubarThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Line width is too thin"),
@@ -3717,7 +3717,7 @@ DRCAll (void)
             SET_FLAG (flag, arc);
             DrawArc (layer, arc);
             drcerr_count++;
-            SetThing (ARC_TYPE, layer, arc, arc);
+            SetFubarThing (ARC_TYPE, layer, arc, arc);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Arc width is too thin"),
@@ -3762,7 +3762,7 @@ DRCAll (void)
             SET_FLAG (flag, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetThing (PIN_TYPE, element, pin, pin);
+            SetFubarThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin annular ring too small"),
@@ -3794,7 +3794,7 @@ DRCAll (void)
             SET_FLAG (flag, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetThing (PIN_TYPE, element, pin, pin);
+            SetFubarThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin drill size is too small"),
@@ -3837,7 +3837,7 @@ DRCAll (void)
             SET_FLAG (flag, pad);
             DrawPad (pad);
             drcerr_count++;
-            SetThing (PAD_TYPE, element, pad, pad);
+            SetFubarThing (PAD_TYPE, element, pad, pad);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pad is too thin"),
@@ -3882,7 +3882,7 @@ DRCAll (void)
             SET_FLAG (flag, via);
             DrawVia (via);
             drcerr_count++;
-            SetThing (VIA_TYPE, via, via, via);
+            SetFubarThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via annular ring too small"),
@@ -3914,7 +3914,7 @@ DRCAll (void)
             SET_FLAG (flag, via);
             DrawVia (via);
             drcerr_count++;
-            SetThing (VIA_TYPE, via, via, via);
+            SetFubarThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via drill size is too small"),
@@ -3959,7 +3959,7 @@ DRCAll (void)
             SET_FLAG (flag, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetThing (LINE_TYPE, layer, line, line);
+            SetFubarThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Silk line is too thin"),
@@ -4011,7 +4011,7 @@ DRCAll (void)
             SET_FLAG (flag, element);
             DrawElement (element);
             drcerr_count++;
-            SetThing (ELEMENT_TYPE, element, element, element);
+            SetFubarThing (ELEMENT_TYPE, element, element, element);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
